.eslintrc.json

```diff
@@ -52,22 +52,40 @@
     "import/no-extraneous-dependencies": [
       "error",
       {
-        "devDependencies": [
-          "**/stories/**/*.*",
-          "**/.storybook/**/*.*"
-        ],
+        "devDependencies": ["**/stories/**/*.*", "**/.storybook/**/*.*"],
         "peerDependencies": true
       }
     ]
   },
   "overrides": [
     {
-      "files": "tests/**/*.ts",
+      "files": ["**/*.test.{ts,tsx,js,jsx}", "**/__tests__/**/*.{ts,tsx,js,jsx}"],
       "env": {
         "node": true,
-        "jest": true
+        "jest": true,
+        "jest/globals": true
+      },
+      "plugins": ["jest", "no-snapshot-testing", "testing-library"],
+      "extends": [
+        "airbnb",
+        "airbnb-typescript",
+        "prettier",
+        "plugin:@cspell/recommended",
+        "plugin:jest/recommended",
+        "plugin:testing-library/react"
+      ],
+      "rules": {
+        // https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-node-access.md
+        "testing-library/no-node-access": ["error", { "allowContainerFirstChild": true }],
+        // https://github.com/VicJer/eslint-plugin-no-snapshot-testing
+        "no-snapshot-testing/no-snapshot-testing": "error",
+        "import/no-extraneous-dependencies": "off",
```

Comment:
Can we scope this to `devDependencies`?

```
"import/no-extraneous-dependencies": [
      "error",
      {
        "devDependencies": [
          "**/stories/**/*.*",
          "**/.storybook/**/*.*",
          "spec/**/*.*"
        ],
        "peerDependencies": true
      }
    ]
```

Reply:
But even in tests we don't need to import something that we didn't include in `package.json`, no? We want to use stuff from 'devDependencies' in tests but not something that we didn't include in any of the dependencies